docs/getting_started/quickstart.md

```diff
@@ -50,4 +54,4 @@ DesanitizeResponse(desanitized_text='Sarah Jane and John Smith will be meeting i
 
 ## Using OpaquePrompts with LangChain
 
-OpaquePrompts offers a [LangChain](https://python.langchain.com/docs/get_started/introduction.html) integration, enabling you to easily build privacy-preserving LLM applications. See the [OpaquePrompts page in the LangChain documentation](https://python.langchain.com/docs/integrations/llms/promptguard) for more.
\ No newline at end of file
+OpaquePrompts offers a [LangChain](https://python.langchain.com/docs/get_started/introduction.html) integration, enabling you to easily build privacy-preserving LLM applications. See the [OpaquePrompts page in the LangChain documentation](https://python.langchain.com/docs/integrations/llms/promptguard) for more.
```

Comment:
https://python.langchain.com/docs/integrations/llms/promptguard -> https://python.langchain.com/docs/integrations/llms/opaqueprompts

Reply:
Thanks for catching this